pkgs/development/python-modules/fastparquet/default.nix

```diff
@@ -1,27 +1,59 @@
-{ lib, buildPythonPackage, fetchFromGitHub, numba, numpy, pandas, pytestrunner
-, thrift, pytestCheckHook, python-snappy, lz4, zstandard, zstd }:
+{ lib
+, buildPythonPackage
+, cramjam
+, fetchFromGitHub
+, fsspec
+, numba
+, numpy
+, pandas
+, pytest-runner
+, thrift
+, pytestCheckHook
+, python-snappy
+, lz4
+, zstandard
+, zstd
+}:
 
 buildPythonPackage rec {
   pname = "fastparquet";
-  version = "0.5.0";
+  version = "0.6.3";
 
   src = fetchFromGitHub {
     owner = "dask";
     repo = pname;
     rev = version;
-    sha256 = "17i091kky34m2xivk29fqsyxxxa7v4352n79w01n7ni93za6wana";
+    sha256 = "05lfviijbd403gii4rjxznrkcqslv8akxbn11b4w0wxvllz7l8n1";
   };
 
-  nativeBuildInputs = [ pytestrunner ];
-  propagatedBuildInputs = [ numba numpy pandas thrift ];
-  checkInputs = [ pytestCheckHook python-snappy lz4 zstandard zstd ];
+  nativeBuildInputs = [
+    pytest-runner
+  ];
+
+  propagatedBuildInputs = [
+    cramjam
+    fsspec
+    lz4
+    numba
+    numpy
+    pandas
+    python-snappy
+    thrift
+    zstandard
+    zstd
```

Comment:
This is a lot of packages to propagate now. Upstream only lists five in its `requirements.txt`. Am I missing something?